packages/discovery-react-components/src/components/SearchFacets/utils/searchFilterTransform.tsx

```diff
@@ -67,36 +69,40 @@ export class SearchFilterTransform {
 
   static toString(facets: SearchFilterFacets): string {
     const fieldFilters = this.fieldsToString(facets.filterFields);
-    const dynamicFilters = this.quoteSelectedFacets(facets.filterDynamic, 'text').join(',');
+    const dynamicFilters = this.escapeSelectedFacets(facets.filterDynamic, 'text').join(',');
     return [fieldFilters, dynamicFilters].filter(Boolean).join(',');
   }
 
-  private static unquoteString(quotedString: string): string {
-    return quotedString.replace(/^"(.+)"$/, '$1').replace(/\\"/, '"');
+  private static unescapeString(escapedString: string): string {
+    const unescapedString = escapedString
+      .replace(/^\((.+)\)$/, '$1')
+      .replace(RegExp(SearchFilterTransform.ESCAPED_CHAR), '$1');
+    return unescapedString;
   }
 
   static fieldsToString(facets: InternalQueryTermAggregation[]): string {
     const filterStrings: string[] = [];
     facets.forEach(facet => {
       const field = get(facet, 'field', '');
       const results = get(facet, 'results', []);
-      const keys = this.quoteSelectedFacets(results, 'key');
+      const keys = this.escapeSelectedFacets(results, 'key');
       if (keys.length) {
-        filterStrings.push(keys.map(key => `${escapeFieldName(field)}::${key}`).join('|'));
+        filterStrings.push(keys.map(key => `${escapeFieldName(field)}:${key}`).join('|'));
       }
     });
     return filterStrings.join(',');
   }
 
-  private static quoteSelectedFacets(
+  private static escapeSelectedFacets(
     facets: (SelectableQueryTermAggregationResult | SelectableDynamicFacets)[],
     key: string
   ): string[] {
+    const escapeRegex = RegExp(SearchFilterTransform.CHARS_TO_ESCAPE);
     return facets
       .filter(result => result.selected)
       .map(result => {
         const text = get(result, key, '');
-        return `${text.replace(/"/, '\\"')}`;
+        return `(${text.replace(escapeRegex, '\\$&')})`;
```

Comment:
Why the parentheses?

Reply:
Ah, this seems to be:

> group search phrases with parens to prevent query ambiguity

Add a comment about this, since it's not obvious from code what the parentheses are meant to do.

Reply:
We used to use double quotes, but that caused the query to be treated as a [phrase query](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-query-operators#phrase). Fucheng recently [removed these quotes](https://github.com/watson-developer-cloud/discovery-components/commit/972d1b76be4d05661599e5f7d22ec629ba8cc7ef). However, in discussions with Mauricio, he mentioned that he uses parentheses to prevent ambiguity within his queries (e.g. the `Google|IBM` example [here](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-query-operators#or)).